components/src/dynamo/vllm/multimodal_utils/request_processor.py

```
@@ -0,0 +1,543 @@
+                    transport,
+                )
+                return None
+
+            results = await receiver.receive(metadata)
+            pickled_items = results.get("__pickled_kwargs_item__")
+            if not pickled_items:
+                return None
+
+            kwargs_items = []
+            for payload in pickled_items:
+                item = pickle.loads(payload)
```

Ruff (0.15.20)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)